src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Active areas of Ethereum research
+description: Explore different areas of open research and learn how to get involved.
+sidebar: true
+lang: en
+---
+
+# Open areas of Ethereum research
+
+One of the primary strengths of Ethereum is that it is constantly being improved by an active research and engineering community. There are many enthusiastic, skilled people worldwide that would like to apply themselves to outstanding issues in Ethereum, but it is not always easy to find out what those issues are. This page aims to outline some of the key areas of active research as a rough guide to Ethereum's cutting edge.
+
+## General research resources
+
+Regardless of the specific topic, there is a wealth of information on Ethereum research to be found at [ethresear.ch](https://ethresear.ch) and the [Eth R&D Discord channel](https://discord.gg/n7JxAeRu). These are the primary places where Ethereum researchers discuss the latest ideas and development opportunities.
+
+## Scaling and Performance
+
+There are ongoing efforts to scale Ethereum using a variety of techniques including sharding the blockchain, danksharding and using rollups. Introductory information on scaling Ethereum is available on our [scaling page](/developers/docs/scaling).
+
+### Layer 2
+
+There are now several Layer 2 protocols that scale Ethereum using different techniques for batching transactions and securing them on Ethereum layer 1. This is a very rapidly growing topic with a lot of research and development potential.
+
+#### Links
+
+[Our Layer-2 page](https://ethereum.org/en/layer-2/)
+
+[EthHub Layer-2](https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/zk-rollups/)
+
+[ethresear.ch Layer 2](https://ethresear.ch/c/layer-2/32)
+
+
+### Bridges
+
+One particular area of Layer-2 that requires more research and development is safe and performant bridges. This includes bridges between various Layer-2s and bridges between Layer 1 and Layer 2. This is a particularly important area of researchg because bridges are commonly targeted by hackers.
+
+#### Links
+
+[Our Bridges page](https://ethereum.org/en/developers/docs/bridges)
+
+[Vitalik on bridges](https://old.reddit.com/r/ethereum/comments/rwojtk/ama_we_are_the_efs_research_team_pt_7_07_january/hrngyk8/)
+
+
+### Sharding
+
+Sharding Ethereum's blockchain has long been part of the development roadmap. However, new scaling solutions such as "Danksharding" are currently taking center stage. 
+
+#### Links
+
+[Our Sharding page](/upgrades/shard-chains/)
```

Comment:
```suggestion
[Introduction to Ethereum sharding](/upgrades/shard-chains/)
```
In general I'd like to avoid vague descriptions like "Our Sharding page". We style internal & external links differently, so users will be able to tell what pages are "ours". I'd focus on explaining what the content of the page is.